### ISSUE TITLE
only process code, ignore strings or comments

### ISSUE THREAD

**Issue by maksym-arutyunyan** [lusingander/cargo-sort-derives#7]
This code should not be modified, because it's not the actual code, its a string

```rs
    config.type_attribute(
        "ic_nervous_system.pb.v1.Percentage",
        "#[derive(PartialOrd, Ord)]",
    );
```

same for commented code
```rs
#[derive(A, B)]  // Don't use #[derive(E, D)]!
```

It may be even harder for multiline strings & comments.
A workaround for ignoring tricky cases can be keyword, eg `sort-derive: ignore-line`, `sort-derive: ignore-block`, `sort-derive: ignore-file`.

**Comment by lusingander** [comment]
Yes, currently it's just a string replacement so of course it works that way.

> It may be even harder for multiline strings & comments.
> A workaround for ignoring tricky cases can be keyword, eg sort-derive: ignore-line, sort-derive: ignore-block, sort-derive: ignore-file.

As you say, we need to consider the implementation policy.
